_overviews/scala3-contribution/procedures-inspection.md

```diff
@@ -13,7 +13,9 @@ while the compiler is running, and inspect produced artifacts of the compiler.
 ## Inspecting variables in-place
 
 Frequently you will need to inspect the content of a particular variable.
-Often, it is sufficient to use `println`.
+You can either use `println`s or the debugger, more info on how to setup the latter [here](https://docs.scala-lang.org/scala3/guides/contribution/procedures-debugging.html).
```

Comment:
It is best practice to use the `link` feature of Jekyll so that we know at build-time if a link is incorrect:

```suggestion
You can either use `println`s or the debugger, more info on how to setup the latter [here]({% link _overviews/scala3-contribution/procedures-debugging.md %}).
```